content/docs/abstract_machine.mdz

```diff
@@ -113,48 +116,59 @@ O - Opcode bits
 +----+----+----+----+
 ```)
 
-8 bits for the opcode leaves 24 bits for the payload, which may or may not be utilized.
-There are a few instruction variants that divide these payload bits.
-
-@ul{@li{0 arg - Used for noops, returning nil, or other instructions that take no
-        arguments. The payload is essentially ignored.}
-    @li{1 arg - All payload bits correspond to a single value, usually a signed or unsigned integer.
-        Used for instructions of 1 argument, like returning a value, yielding a
-        value to the parent fiber, or doing a (relative) jump.}
-    @li{2 arg - Payload is split into byte 2 and bytes 3 and 4. The first argument is the 8 bit value from
-        byte 2, and the second argument is the 16 bit value from bytes 3 and 4 (@code[instruction >> 16]). Used for instructions of
-        two arguments, like move, normal function calls, conditionals, etc.}
-    @li{3 arg - Bytes 2, 3, and 4 each correspond to an 8 bit argument.
+Eight bits for the opcode leaves 24 bits for the payload, which may or may not
```

Comment:
From a general writing perspective, AFAIK, writing "eight" instead of "8" here is correct, but I wonder if it is easier to reason about if "8" is kept here? I mean, in my head "8+24" is more straight-forward than "Eight+24".

Reply:
Maybe start with the adjective (first, rightmost) and use number 8?

Reply:
I believe traditional style guides in English (such as those used by newspapers) will provide that the numbers from 1-10 should be written out as words (i.e. 'one', 'two'... etc) before being replaced with numerals (i.e. '11', '12'... etc). An exception to this rule is if the number begins a sentence in which case it should always be written out as words.

Generally, I think I avoided making changes because, as a technical document, it seems to me reasonable that numerals be used for all numbers. I honestly don't even remember if I made this change intentionally or whether it was just 'muscle' memory :P I'm happy to go back to numerals. I think they're easier for people to scan and, as noted above, are appropriate for a document aimed at a technical audience.

Reply:
Changed this to be `Using 8 bits...`